src/Compilers/Core/Portable/CommandLine/EditorConfig.cs

```diff
@@ -31,7 +31,7 @@ internal sealed class EditorConfig
         /// not be removed.
         /// </remarks>
         public static ImmutableHashSet<string> ReservedKeys { get; }
-            = ImmutableHashSet.CreateRange(CaseInsensitiveComparison.Comparer, new[] {
+            = ImmutableHashSet.CreateRange(Section.PropertiesKeyComparer, new[] {
```

Comment:
Note: ReservedValues below isn't using the `PropertiesKeyComparer` because 1) it isn't a key but mainly 2) the spec is quite vague on this and there really isn't such a thing as a "reserved value". It's something I made up to make certain values explicitly mentioned in the docs case-insensitive, and others case-sensitive.